10-JavaScript/testing.md

```diff
@@ -23,96 +22,119 @@ In short, most interesting JavaScript code is full of *external dependencies*. M
+Because Jest is a behavior-driven testing language, testing using Jest for JavaScript is fairly similar to what we saw using Minitest for Ruby and Rails. As always, there are some specific differences.
 
 Each test should have these components to describe the test behavior.
 
 #### `describe` Blocks
-Describe blocks should create test groupings based on _objects_ and _functions_.
+Describe blocks should create test groupings based on _objects_ and _functions_.  They are optional, but provide some readability and organization.  Like Minitest you can also nest `describe` blocks.
 
-We should use one `describe` block for our overall `Word` model, and another nested one for each method or set of behavior we want to test. `Word` has one for the constructor, another for validations and a third for the `score()` method.
+We will use one `describe` block for our overall `Pangram` class, and another nested one for the `isPangram` method.
 
 Each `describe` function has two parameters. The first is the description of the `describe` block and the second is the function which contains the actions/logic. Note that we add a `;` at the end of each `describe` block.
 
 ```javascript
-describe('Word', () => {
-  describe('constructor', () => {
+describe('Pangram class',  () =>  {
+
+
+  describe('isPangram()', () => {
 
   });
 });
 ```
 
-#### `it` Blocks
+#### `test` Blocks
 
-`it` blocks should define one distinct test. The description that goes along with the `it` block should describe the specific scenario that you are testing.
+`test` blocks should define one distinct test. The description that goes along with the `test` block should describe the specific scenario that you are testing.
 
 ```javascript
-describe('Word', () => {
-  describe('constructor', () => {
-    it ('Converts text to lowercase', () => {
+describe('Pangram class',  () =>  {
+  test('Pangram is defined', function() {
+
+  }
+
+  describe('isPangram()', () => {
 
-    });
   });
 });
 ```
 
+Similar to what you have in Minitest you can use `it`, in place of `test` if `it` (hic) makes you feel more comfortable, but the Jest documentation uses `test`, and so Ada's examples will as well.
+
 #### `expect` Statements
 
 Expectations should be the "meat and potatoes" of your tests, inside of your `it` blocks. Each test has at least one `expect` statement to ensure the behavior is as expected.
 
-The syntax of `describe` and `it` is pretty similar to Minitest (at least as similar as Ruby and JS can be), but `expect` is somewhat different. Note that there are actually two functions being called, the `expect` and the matcher (`toEqual` in this case), each taking one argument.
+The syntax of `describe` and `test` is pretty similar to Minitest (at least as similar as Ruby and JS can be), but `expect` is somewhat different. Note that there are actually two functions being called, the `expect` and the matcher (`toBeDefined` in this case).
```

Comment:
Instead of this sentence with the "Note", can you explicitly state how the `expect` statement is constructed? (I know that you're pulling from the notes as they previously existed, but I feel like this could use a more explicit explanation)